AaxDecrypter/AaxcDownloadConverter.cs

```diff
@@ -51,6 +53,11 @@ public AaxcDownloadConverter(string outFileName, string cacheDirectory, Download
             downloadLicense = ArgumentValidator.EnsureNotNull(dlLic, nameof(dlLic));
             OutputFormat = outputFormat;
 
+            if (Serilog.Log.Logger.IsVerboseEnabled())
```

Comment:
No conditional required

Reply:
Ok. I was just copying the logging pattern you used elsewhere.